Fix T970871 and remove encodeHtml from docs (#2180)
DevExpress/devextreme-documentation: concepts/Common/Security Considerations/20 HTML Encoding/30 Potentially Dangerous API Members/0 Potentially Dangerous API Members.md

> DevExtreme components encode HTML in most cases. However, several components include API members that allow you to disable or bypass the encoding. The following sections describe these potentially vulnerable API members.

Reviewer: "DevExtreme components encode HTML in most cases." - This is a vague statement since we don't specify the cases.

```suggestion
Several components include API members that allow you insert unencoded HTML. The following sections describe these potentially vulnerable API members.
```

> DevExtreme components encode HTML in most cases. However, several components include API members that allow you to disable or bypass the encoding. The following sections describe these potentially vulnerable API members.

Reviewer: I'm not sure about the word "dangerous". It looks very serious :) How about "vulnerable"? Also, we could drop the word "members" to make the title shorter.
